codewof/tests/programming/test_views.py

```diff
@@ -113,13 +111,6 @@ def test_get_object_question_does_not_exist(self):
         resp = self.client.get('/questions/{}/'.format('fake-primary-key'))
         self.assertEqual(resp.status_code, 404)
 
-    def test_get_object_question_not_in_study(self):
```

Comment:
I'm sure you already know this and were just trying to get this done asap but remember to write tests for the new stuff as well when you have time :)